src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -74,37 +74,145 @@ public DevBoxInstance(IDevBoxAuthService devBoxAuthService)
+        var webUrl = new Uri(json.GetProperty("webUrl").ToString());
+        return (webUrl, remoteUri);
+    }
+
+    public ComputeSystemOperations SupportedOperations =>
+        ComputeSystemOperations.Start | ComputeSystemOperations.ShutDown | ComputeSystemOperations.Delete;
 
     public bool IsValid
     {
         get;
         private set;
     }
 
-    private IAsyncOperation<ComputeSystemOperationResult> PerformRESTOperation(string operation) => throw new NotImplementedException();
+    private IAsyncOperation<ComputeSystemOperationResult> PerformRESTOperation(string operation, HttpMethod method)
+    {
+        return Task.Run(async () =>
+        {
+            var api = $"{BoxURI}:{operation}?api-version=2023-04-01";
+            Log.Logger()?.ReportInfo($"Starting {Name} with {api}");
+
+            var httpClient = _authService.GetDataPlaneClient(DevId);
+            if (httpClient == null)
+            {
+                var ex = new ArgumentException("PerformRESTOperation: HTTPClient null");
+                return new ComputeSystemOperationResult(ex, string.Empty, string.Empty);
+            }
+
+            var response = await httpClient.SendAsync(new HttpRequestMessage(method, api));
+            if (response.IsSuccessStatusCode)
+            {
+                var res = new ComputeSystemOperationResult("Success");
+                return res;
+            }
+            else
+            {
+                var ex = new HttpRequestException($"PerformRESTOperation: {operation} failed. {response.StatusCode} {response.ReasonPhrase}");
+                return new ComputeSystemOperationResult(ex, string.Empty, string.Empty);
+            }
+        }).AsAsyncOperation();
+    }
 
     public IAsyncOperation<ComputeSystemOperationResult> Start(string options)
     {
-        return PerformRESTOperation("start");
+        return PerformRESTOperation("start", HttpMethod.Post);
     }
 
     public IAsyncOperation<ComputeSystemOperationResult> ShutDown(string options)
     {
-        return PerformRESTOperation("stop");
+        return PerformRESTOperation("stop", HttpMethod.Post);
     }
 
-    public IAsyncOperation<ComputeSystemOperationResult> Connect(string properties) => throw new NotImplementedException();
+    public IAsyncOperation<ComputeSystemOperationResult> Restart(string options)
+    {
+        return PerformRESTOperation("restart", HttpMethod.Post);
+    }
 
-    public IAsyncOperation<ComputeSystemStateResult> GetState(string options) => throw new NotImplementedException();
+    public IAsyncOperation<ComputeSystemOperationResult> Delete(string options)
+    {
+        return PerformRESTOperation("delete", HttpMethod.Delete);
+    }
+
+    public IAsyncOperation<ComputeSystemOperationResult> Connect(string properties)
+    {
+        // Todo: Is there a better way to do this?
+        return Task.Run(() =>
+        {
+            try
+            {
+                var psi = new ProcessStartInfo();

```

Comment:
FYI: ProcessStartInfo does not use ShellExecute by default.  If you need Shell behavior, set [UseShellExecute](https://learn.microsoft.com/dotnet/api/system.diagnostics.processstartinfo.useshellexecute?view=net-8.0) to true.

Reply:
Being set on the next line